### DISCUSSION TITLE
Feat request: Init the SEO fields on creation & implement snippet tags

### DISCUSSION THREAD

**Discussion by matthijs166** [payloadcms/payload-plugin-seo#23] [General]
It would be awesome when there is a new page created that the fields would be populated with initial values from the generators.

It would be hard to implement because there is no slug or title etc set. So for this to work we need some kind of template snippets to bind fields after a page read.

Would suggest to use the same format as Yoast SEO https://yoast.com/help/list-available-snippet-variables-yoast-seo/
And the names could be the name of the field.

**Comment by jacobsfletch** [comment]
@matthijs166 this is a great idea. I think we can come up with some sort of pattern for this.

**Comment by Rar9** [comment]
Tokens would be great if they could be used to pre-populate content when it's created. 

Like xxx best payloadcms 

Where xxx could be anything like collection /category....
